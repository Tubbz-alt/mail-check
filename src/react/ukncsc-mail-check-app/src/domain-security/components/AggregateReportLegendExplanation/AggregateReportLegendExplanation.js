import React from 'react';
import { Header } from 'semantic-ui-react';

export default ({ title, description }) => (
  <React.Fragment>
    <Header as="h3">{title}</Header>
    <p>{description}</p>
  </React.Fragment>
);
