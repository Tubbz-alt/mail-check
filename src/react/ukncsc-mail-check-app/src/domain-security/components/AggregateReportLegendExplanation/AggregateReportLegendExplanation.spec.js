import React from 'react';
import { render } from 'react-testing-library';
import AggregateReportLegendExplanation from './AggregateReportLegendExplanation';

describe('AggregateReportLegendExplanation', () => {
  let container;

  describe('when getting a aggregate report legend explanation', () => {
    beforeEach(() => {
      ({ container } = render(
        <AggregateReportLegendExplanation
          title="Title"
          description="Explanation"
        />
      ));
    });

    test('it should match the snapshot', () =>
      expect(container).toMatchSnapshot());
  });
});
