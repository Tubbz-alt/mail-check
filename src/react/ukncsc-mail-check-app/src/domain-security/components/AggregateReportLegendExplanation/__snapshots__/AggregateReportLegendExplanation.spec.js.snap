// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AggregateReportLegendExplanation when getting a aggregate report legend explanation it should match the snapshot 1`] = `
<div>
  <h3
    class="ui header"
  >
    Title
  </h3>
  <p>
    Explanation
  </p>
</div>
`;
